network/Networking.cpp

```diff
@@ -13,7 +13,11 @@ namespace {
 namespace Networking {
     const std::string DISCOVERY_QUESTION = "Yo, can I play Free-O here, dog?";
     const std::string DISCOVERY_ANSWER = "Word!";
+#ifdef FREEORION_OPENBSD
```

Comment:
What is the point of this condition?  Why does OpenBSD need a different linger time?

Reply:
Understood. Can you add a comment for this on the change?